UPGRADE.md

```diff
@@ -2,6 +2,16 @@
 
 ## dev-develop
 
+### Search
+
+To index multiple fields (and `category_list` content-type) you have to add the attribute `type="array"` to the
+`sulu.search.field` tag. The `tag_list` content-type has its own search-field type `tags`
+(`<tag name="sulu.search.field" type="tags"/>`).
```

Comment:
the categories returns already an id array ... but we handle the tags a lot with names therefor we have to convert them ...